web/templates/confirm_dialog/confirm_deactivate_realm.hbs

```diff
@@ -1,2 +1,26 @@
 <p>{{t "This action is permanent and cannot be undone. All users will permanently lose access to their Zulip accounts." }}</p>
+
+{{#if development_environment}}
+<div class="alert" id="dev_env_msg"></div>
+{{/if}}
+<div class="input-group">
+    <label for="expires_in">{{t "Automatically delete all data after" }}</label>
+    <select id="expires_in" class="deactivated-realm-expires-in bootstrap-focus-style">
+        {{#each delete_in_options}}
+            <option {{#if this.default }}selected{{/if}} name="delete_in" value="{{this.value}}">{{this.description}}</option>
+        {{/each}}
+    </select>
+    <p id="expires_on"></p>
+    <div id="custom-deactivate-realm-expiration-time" class="dependent-settings-block custom-time-input-container">
+        <p class="allowed_duration_message">{{allowed_duration_message}}</p>
+        <label for="expires_in">{{t "Custom time" }}</label>
+        <input type="text" autocomplete="off" name="custom-expiration-time" id="custom-expiration-time-input" class="custom-expiration-time inline-block" value="" maxlength="3"/>
+        <select class="custom-expiration-time bootstrap-focus-style" id="custom-expiration-time-unit">
```

Comment:
Using expiration time here feels weird as we use "expire" for invites and we instead use "delete" for realms. Would be good to rename these and also refactor the common functions to receive the element as parameter instead of using `$("#expires_in")`.

Reply:
There can be two options - either you can pass the containing div of all these elements or you can just pass the main select element `$("#expires_in")` and then to get the custom element you can do something like `$("#expires_in").parent().find({some-common-class-used-for-custom-inputs})`. 

think we use the second approach in custom limit organization settings, so would be good to use same here as well. You can re-check this once to confirm what we do for organization settings.